bin/builder_run

```diff
@@ -12,21 +12,16 @@
 #> bin/builder_run some_command_to_execute
 
 # Usage example:
+#> bin/builder_run pwd
+#> bin/builder_run bin/package_all <version>
 #> bin/builder_run bin/server_build; bin/server_package_debian 0.0.1 amd64
 
-COMMAND=$1
+COMMAND=$@
 if [ -z "$COMMAND" ]; then
     printf "\e[0;31m%s\e[0m\n" "Missing command!"
     exit 1
 fi
 
-printf "\e[0;32m%s\e[0m\n" "running builder command: $COMMAND .." \
-
-docker run --rm -v $(pwd):/go/src/github.com/mysterium/node mysteriumnetwork/mysterium-node-builder -c "${COMMAND}"
-
-if [ $? -ne 0 ] ; then
-    printf "\e[0;31m%s\e[0m\n" "Running builder failed!"
```

Comment:
Why have you removed all `printf`, are they moved somewhere?

Reply:
Remove trace level logging